Ginger/GingerCoreNETUnitTest/SolutionRepositoryLib/RepositoryTest.cs

```diff
@@ -1281,5 +1281,136 @@ public void CopyRepoItem_TrackCopyTest()
             Assert.AreEqual(2, bf.Activities.Count);
             Assert.AreNotSame(bf, copiedItem);
         }
+
+        [TestMethod]
+        public void CopyItem_ChildWithNewGUIDTest()
+        {
+            //Arrange
+            BusinessFlow bf = new BusinessFlow("Test");
+
+            Activity activity = new Activity();
+            activity.ActivityName = "Login";
+
+            ActDummy dumAct = new ActDummy();
+            //Guid sfGuid = Guid.NewGuid();
+            FlowControl sampleFC = new FlowControl()
+            {
+                Active = true,
+                FlowControlAction = eFlowControlAction.GoToAction,
+                ConditionCalculated = "aaa",
```

Comment:
this is not enough- need to set the Selected Action value- which is been done with GUID#Name combination- see how it saved to XML when you doing it from UI and mimic it on unit test- then need to validate the GUID of poined Action to see if it was changed/ unchanged as expected based on Copy with new GUID or copy with same GUID